src/jit/codegenarm64.cpp

```diff
@@ -906,7 +906,7 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 #endif
 
     assert(block != NULL);
-    assert(block->bbFlags && BBF_FUNCLET_BEG);
+    assert(block->bbFlags & BBF_FUNCLET_BEG);
```

Comment:
Nit: would you mind fixing up the `NULL` above and changing the assert you've corrected to `assert((block->bbFlags & BBF_FUNCLET_BEG) != 0)`?